resources/graphics.svg

```diff
@@ -3721,51 +3721,6 @@
          id="rect4320"
          style="fill:#c6c6c6;fill-opacity:1;stroke:#3c3c3c;stroke-width:1.00000024;stroke-linecap:round;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" />
     </g>
-    <g
```

Comment:
I'm not seeing anything looking like an ID collision in this block - can you confirm that deleting it was intentional?

Reply:
Ahh sorry, yeah, it seemed to be a redundant icon, which was causing confusion about which one should be exported. The lower one (removed) didn't seem to have any export identifiers anywhere:

![image](https://user-images.githubusercontent.com/896779/79340063-b412ca00-7f21-11ea-8eaf-b68cfceaf68b.png)